tests/src/Integration/Action/EntityPathAliasCreateTest.php

```diff
@@ -0,0 +1,192 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Tests\rules\Integration\Action\EntityPathAliasCreateTest.
+ */
+
+namespace Drupal\Tests\rules\Integration\Action;
+
+use Drupal\Core\Entity\ContentEntityType;
+use Drupal\Tests\rules\Integration\RulesEntityIntegrationTestBase;
+
+/**
+ * @coversDefaultClass \Drupal\rules\Plugin\Action\EntityPathAliasCreate
+ * @group rules_actions
+ */
+class EntityPathAliasCreateTest extends RulesEntityIntegrationTestBase {
+
+  /**
+   * The action to be tested.
+   *
+   * @var \Drupal\rules\Core\RulesActionInterface
+   */
+  protected $action;
+
+  /**
+   * The mocked alias storage service.
+   *
+   * @var \PHPUnit_Framework_MockObject_MockObject|\Drupal\Core\Path\AliasStorageInterface
+   */
+  protected $aliasStorage;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setUp() {
+    parent::setUp();
+
+    // Prepare a content entity type instance.
+    $this->entityType = new ContentEntityType([
+      'id' => 'test',
+      'label' => 'Test',
+      'entity_keys' => [
+        'bundle' => 'bundle',
+      ],
+    ]);
+
+    // Prepare mocked entity manager.
+    $this->entityManager = $this->getMockBuilder('Drupal\Core\Entity\EntityManager')
+      ->setMethods(['getBundleInfo', 'getDefinitions', 'getBaseFieldDefinitions'])
+      ->setConstructorArgs([
+        $this->namespaces,
+        $this->moduleHandler,
+        $this->cacheBackend,
+        $this->languageManager,
+        $this->getStringTranslationStub(),
+        $this->getClassResolverStub(),
+        $this->typedDataManager,
+        $this->getMock('Drupal\Core\KeyValueStore\KeyValueStoreInterface'),
+        $this->getMock('Symfony\Component\EventDispatcher\EventDispatcherInterface')
+      ])
+      ->getMock();
+
+    // Return a mocked list of base fields definitions.
+    $this->entityManager
+      ->expects($this->any())
+      ->method('getBaseFieldDefinitions')
+      ->will($this->returnValue([]));
+
+    // Return a mocked list of entity types.
+    $this->entityManager
+      ->expects($this->any())
+      ->method('getDefinitions')
+      ->will($this->returnValue(['test' => $this->entityType]));
+
+    // Return some dummy bundle information for now, so that the entity manager
+    // does not call out to the config entity system to get bundle information.
+    $this->entityManager
+      ->expects($this->any())
+      ->method('getBundleInfo')
+      ->with($this->anything())
+      ->will($this->returnValue(['test' => ['label' => 'Test']]));
+
+    $this->container->set('entity.manager', $this->entityManager);
```

Comment:
Everything you are mocking here for the EntityManager already exists in parent::setup().